collections-arrays/ArrayPool/ArrayPool.Benchmark/Benchmark.cs

```diff
@@ -0,0 +1,33 @@
+﻿using BenchmarkDotNet.Attributes;
+using System.Buffers;
+
+namespace ArrayPool.Benchmark;
+
+[MemoryDiagnoser]
+public class Benchmark
+{
+    [Params(100, 1000, 10000, 100000)]
+    private int ArraySize { get; set; }
```

Comment:
Since this is `private`, why is it a property? Could we use a field instead or? Also, let's follow our naming convention where all private fields should start with the underscore.

Reply:
@mahrncic -  The correct type is `public`. That is because the [Params] attribute needs `public` setter, and not `private`. I fixed that in the next PR.